server/api/admin/contact-us.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from 'express';
```

Comment:
For now this is fine, but I think in the future if we're planning on displaying a prettified json file to the cms, then this would require some rework. for example, if we wanted to add a new role to the execs like VP Gaming or something else, this wouldn't work. Ideally, we could just take the contents of the new execs.json and overwrite it with the old one. But this is something for the future, great work so far

Reply:
^ referring to the whole file

Reply:
True, we'd probably need something entirely different when we develop more generic JSON modification.  This definitely isn't the final approach 😅